cli/util/shell-exec.go

```diff
@@ -0,0 +1,72 @@
+package util
+
+import (
+  "bytes"
+  "io/ioutil"
+	"os"
+	"os/exec"
+  "syscall"
+
+	"github.com/fatih/color"
+)
+
+const defaultFailedCode = 1
+
+// Set up the output streams (and colors) to stream command output if verbose is configured
+func StreamCommand(cmd *exec.Cmd) error {
+	return RunCommand(cmd, false)
+}
+
+// Set up the output streams (and colors) to stream command output regardless of verbosity
+func ForceStreamCommand(cmd *exec.Cmd) error {
+	return RunCommand(cmd, true)
+}
+
+// Run the command
+func RunCommand(cmd *exec.Cmd, forceOutput bool) error {
+	cmd.Stderr = os.Stderr
+	if Logger().IsVerbose || forceOutput {
+		cmd.Stdout = os.Stdout
+	} else {
+		cmd.Stdout = ioutil.Discard
+	}
+
+	color.Set(color.FgCyan)
+	err := cmd.Run()
+	color.Unset()
+	return err
+}
+
+func PassthruCommand(cmd *exec.Cmd) (stdout string, stderr string, exitCode int) {
```

Comment:
Yes I can. This function is not yet working as intended so I'll be back to it :)
It started (pretty obviously) with http://stackoverflow.com/a/40770011/38408